CSS/style.css

```diff
@@ -0,0 +1,902 @@
+body {
+  background-color: #f4f5f7;
+  width: 100vw;
+  margin: 0;
+  font-family: 'Poppins', sans-serif;
+}
+
+header {
+  width: 100vw;
+  position: relative;
+  margin: 0;
+  display: flex;
+  align-items: center;
+  background-color: white;
+}
+
+#logo {
+  text-decoration: none;
+  color: #6070ff;
+  margin: 24px auto 24px 20px;
+  width: 100vm;
+  height: 20px;
+  font-size: 18px;
+  display: flex;
+  font-weight: 700;
+}
+
+p,
+a {
+  font-size: larger;
+}
+
+h1,
+h2 {
+  font-size: x-large;
+}
+
+.nav-pic {
+  height: 12px;
+  padding-right: 15px;
+}
+
+.nav-ul {
+  list-style: none;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  height: 15px;
+}
+
+.bar {
+  display: block;
+  width: 25px;
+  height: 3px;
+  margin: 5px auto;
+  -webkit-transition: all 0.3s ease-in-out;
+  transition: all 0.3s ease-in-out;
+  background-color: #000;
+}
+
+form li {
+  margin-bottom: 20px;
+}
+
+.nav-ul li {
+  display: flex;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 20px;
+  align-items: center;
+  text-align: center;
+  color: #344563;
+  margin-left: 32px;
+}
+
+.nav-ul a {
+  text-decoration: none;
+  transition: transform 0.2s;
+}
+
+.home {
+  background-image: url("../img/background.png");
+  background-color: white;
+  background-size: cover;
+  border-bottom-left-radius: 70px;
+  background-repeat: no-repeat;
+  display: flex;
+  flex-direction: column;
+  padding: 50px 24px;
+  padding-right: 0;
+}
+
+.home h1 {
+  font-size: 40px;
+  margin-bottom: 0;
+  font-weight: 900;
+  line-height: 52px;
+  font-family: 'Poppins', sans-serif;
+}
+
+.home p {
+  width: 270px;
+  font-size: 16px;
+}
+
+.home-content {
+  /* margin: 0% 0% 0% 8%; */
+  color: #172b4d;
+}
+
+.home-content p {
+  color: #172b4d;
+}
+
+.connect p {
+  color: #7f8cff;
+}
+
+.home li {
+  padding-right: 20px;
+}
+
+form > ul {
+  list-style: none;
+  padding-left: 0;
+}
+
+.home > ul {
+  list-style: none;
+  display: flex;
+  padding-left: 0;
+}
+
+.work {
+  display: grid;
+  align-items: center;
+  padding: 114px 24px;
+  gap: 88px;
+  left: 0;
+  top: 100px;
+}
+
+.work.active {
+  filter: blur(50px);
+}
+
+.grid-item {
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  background-color: white;
+  padding: 15px 15px 15px 15px;
+  border: 1px solid #dfe1e6;
+  border-radius: 16px;
+  width: 75%;
+  margin: auto;
+}
+
```

Comment:
- [ ] From the project requirements, you should stick to the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122&t=Tt1vCN6lfPrkflxz-0), But as you can see there are some differences between the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122&t=Tt1vCN6lfPrkflxz-0) and your design
    - Your cards margins and padding do not match the Figma design
    - The style of the `company`, `role`, and `year` does not the Figma design
    - The color of the text in the cards does not match the design
    
    | Your Design |Figma Design |
    | :---:   | :---: | 
    | ![image](https://user-images.githubusercontent.com/92303891/227723785-a1867c3a-2770-4744-ab53-8ce45cbb6ddb.png) | ![image](https://user-images.githubusercontent.com/92303891/227723530-ec859909-aa5d-4399-80d1-835e6a9f6f0d.png) |